sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretClient.cs

```diff
@@ -139,6 +139,76 @@ public virtual Response<KeyVaultSecret> GetSecret(string name, string version =
             }
         }
 
+#pragma warning disable AZC0015 // Unexpected client method return type.
```

Comment:
>AZC0015 [](http://example.com/codeflow?start=24&length=7)

Is this coming from our analyzers?

Reply:
Yes, although it should be [fixed now ](https://github.com/Azure/azure-sdk-tools/pull/4512)- we just need to update the version reference to the latest analyzer

Reply:
Updated. @christothes I also opened PR #33817 to update it in main since I'll hold on this PR until after we GA.